.github/workflows/comment-wheels-pr.yml

```diff
@@ -12,50 +12,13 @@ jobs:
       - uses: actions/github-script@v7
         with:
           script: |
-            async function insertUpdateComment(owner, repo, issue_number, purpose, body) {
-                const {data: comments} = await github.rest.issues.listComments(
-                    {owner, repo, issue_number}
-                );
-                const marker = `<!-- bot: ${purpose} -->`;
-                body = marker + "\n" + body;
-                const existing = comments.filter((c) => c.body.includes(marker));
-                if (existing.length > 0) {
-                    const last = existing[existing.length - 1];
-                    core.info(`Updating comment ${last.id}`);
-                    await github.rest.issues.updateComment({
-                        owner, repo,
-                        body,
-                        comment_id: last.id,
-                    });
-                } else {
-                    core.info(`Creating a comment in issue / PR #${issue_number}`);
-                    await github.rest.issues.createComment({issue_number, body, owner, repo});
-                }
-            }
+            import { insertUpdateComment, getArtifacts } from 'comment-utils.js';
 
-            const {owner, repo} = context.repo;
-            const run_id = ${{github.event.workflow_run.id}};
-
-            // Find the PR with the right sha, see https://github.com/orgs/community/discussions/25220
-            const response = await github.rest.search.issuesAndPullRequests({
-                q: 'repo:${{ github.repository }} is:pr sha:${{ github.event.workflow_run.head_sha }}',
-                per_page: 1,
+            let artifacts = getArtifacts({
+                repo: ${{ github.repository }},
```

Comment:
the function takes multiple arguments but you are calling it with a single object. I think this will fail to run